{"name": "Ошибка, при создании курьера с занятым логином", "status": "passed", "start": 1717539603897, "stop": 1717539604304, "uuid": "14e797e6-395e-4d2a-a405-c9d48081a479", "historyId": "9c1462b9d23eb2fd2dc889d029ad3343", "testCaseId": "9c1462b9d23eb2fd2dc889d029ad3343", "fullName": "tests.test_create_courier.TestCreateCourier#test_create_courier_with_existing_login_error_409", "labels": [{"name": "story", "value": "Создание курьера"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "Rayless"}, {"name": "thread", "value": "25356-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}